zilencer/management/commands/populate_db.py

```diff
@@ -361,6 +361,7 @@ def handle(self, *args: Any, **options: Any) -> None:
                 ("Polonius", "polonius@zulip.com"),
                 ("Desdemona", "desdemona@zulip.com"),
                 ("शिव", "shiva@zulip.com"),
+                ("Shylock", "shylock@zulip.com"),
```

Comment:
Is this name fine? (We do have a user with same name in the analytics database).